helm_tests/airflow_aux/test_pod_template_file.py

```diff
@@ -98,6 +98,7 @@ def test_should_add_an_init_container_if_git_sync_is_true(self):
             "securityContext": {"runAsUser": 65533},
             "image": "test-registry/test-repo:test-tag",
             "imagePullPolicy": "Always",
+            "envFrom": [],
```

Comment:
Can you also write tests where we override it?

Reply:
@rasulkarimov can you address the above comment?

Reply:
Updated the "extraEnvFrom" variables and "envFrom" asserts in helm_test.
Can someone trigger the test pipelines?